internals-js/src/error.ts

```diff
@@ -359,6 +359,26 @@ const SATISFIABILITY_ERROR = makeCodeDefinition(
   'Subgraphs can be merged, but the resulting supergraph API would have queries that cannot be satisfied by those subgraphs.',
 );
 
+const OVERRIDE_FROM_SELF_ERROR = makeCodeDefinition(
+  'OVERRIDE_FROM_SELF_ERROR',
+  'Field with `@override` directive has "from" location that references its own subgraph.',
+);
+
+const OVERRIDE_SOURCE_HAS_OVERRIDE = makeCodeDefinition(
+  'OVERRIDE_SOURCE_HAS_OVERRIDE',
+  'Field which is overridden to another subgraph is also marked @override.',
+);
+
+const OVERRIDE_ON_BOTH_FIELD_AND_TYPE = makeCodeDefinition(
```

Comment:
It's not a huge deal, but I'll note that this error code is currently unused since we don't yet support override on types, but the code still currently shows in the generated doc. I understand that we want override on types as a fast followup, but I've been burned by "guaranteed-fast-followup" a few too many times and my preference would be to remove this for now.